Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -7,20 +7,23 @@
         'Geometry/CMSCommonData/data/materials.xml',
         'Geometry/CMSCommonData/data/rotations.xml',
         'Geometry/CMSCommonData/data/extend/v2/cmsextent.xml',
-        'Geometry/CMSCommonData/data/cms/2019/v2/cms.xml',
+#        'Geometry/CMSCommonData/data/cms/2019/v2/cms.xml',
```

Comment:
please delete all commented-out lines in this file

Reply:
> please delete all commented-out lines in this file

done